### DISCUSSION TITLE
Why github actions builds the petstore application and produce artifact when building is included in Dockerfile itself.

### DISCUSSION THREAD

**Discussion by kavipriyanjeevanandam** [chtrembl/azure-cloud#38] [Q&A]
Why github actions builds the petstore application and produce artifact when building is included in Dockerfile itself. Also the README file says Docker is not setup for multistage build but multistage build is present in the Dockerfile

**Comment by chtrembl** [comment]
Great question @kavipriyanjeevanandam I just updated the guide, the multi stage Docker build was introduced after the 1st cut of the guide, and I left this artifact upload in their for academic purposes to show the features of the upload-artifact action, feel free to remove it and the guide is now updated.
